docs/release-notes.rst

```diff
@@ -3,6 +3,45 @@ Release notes
 
 This document describes major installer-related changes in Fedora releases.
 
+Fedora 38
+#########
+
+General changes
+---------------
+
+Faster core dumps
+^^^^^^^^^^^^^^^^^
+
+Previously, Anaconda used a custom setup for handling tracebacks and saving core dumps. This is
+now realized by using the ``faulthandler`` Python module and the ``systemd-coredump`` service.
+As a result, the same debugging data is still available, while the installation environment
```

Comment:
At least to me using plural seems to be better way. `data are` instead of `data is` I know both are acceptable so I'm not forcing the change.

Reply:
You're right - I had to think about this a bit, but then realized this was one thing that I had to re-learn according the IBM style guide when I started writing docs some years ago.